.travis.yml

```diff
@@ -49,6 +49,7 @@ env:
     - APP_CONFIG_AUTH_TYPE=BASIC
     - APP_CONFIG_OAUTH2_HOST=http://localhost:4200/auth/realms/alfresco
     - APP_CONFIG_OAUTH2_CLIENTID=alfresco
+    - APP_CONFIG_PLUGIN_AOS=true
```

Comment:
are we sure that the default value should be true ? ( just asking )

Reply:
Yes, by default the AOS plugin is already enabled. The goal of this issue is to add the possibility of disabling it.